auth-api/tests/unit/api/test_entity.py

```diff
@@ -486,3 +486,16 @@ def test_get_entity_contacts(client, jwt, session):
     assert data['email'] == 'fo*@ba*****'
     assert 'phone' not in data
     assert 'phone_extension' not in data
+
+
+def test_get_entity_authentication(client, jwt, session):
+    """Assert that an entity authentication can be retrieved."""
+    headers = factory_auth_header(jwt=jwt, claims=TestJwtClaims.public_user_role)
+    rv = client.get(f'/api/v1/entities/{TestEntityInfo.entity1["businessIdentifier"]}/authentication',
```

Comment:
You will probably need to create the entity first, should be able to reference other tests for this

Reply:
existing test has it:

```
def test_get_entity_contacts(client, jwt, session):
    """Assert that an entity contacts can be retrieved."""
    headers = factory_auth_header(jwt=jwt, claims=TestJwtClaims.system_role)
    rv_create = client.post('/api/v1/entities', data=json.dumps(TestEntityInfo.entity1),
                            headers=headers, content_type='application/json')
    assert rv_create.status_code == http_status.HTTP_201_CREATED
    client.post('/api/v1/entities/{}/contacts'.format(TestEntityInfo.entity1['businessIdentifier']),
                headers=headers, data=json.dumps(TestContactInfo.contact1), content_type='application/json')

    headers = factory_auth_header(jwt=jwt, claims=TestJwtClaims.public_user_role)
    rv = client.get(f'/api/v1/entities/{TestEntityInfo.entity1["businessIdentifier"]}/contacts',
                    headers=headers, content_type='application/json')
    assert rv.status_code == http_status.HTTP_200_OK
    data = json.loads(rv.data)
    assert data['email'] != TestContactInfo.contact1['email']
    assert data['email'] == 'fo*@ba*****'
    assert 'phone' not in data
    assert 'phone_extension' not in data
    ```